sdk/resourcemanager/azure-resourcemanager-monitor/CHANGELOG.md

```diff
@@ -2,13 +2,12 @@
 
 ## 2.18.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- Removed unused classes.
+- Removed `JsonFlatten` annotation from model classes.
+- Added `final` modifier to model classes previously annotated with `JsonFlatten`.
+- Changed type of `CategoryType` from enum to class.
```

Comment:
This is a breaking change in convenience layer, currently no idea on how to deal with it...